### PR TITLE
Dx 499

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-499.s3-website-us-east-1.amazonaws.com/

File: accountCredentials.md [hunk 1]
```diff
@@ -20,3 +13,37 @@ Specific information for finding your credentials can be found [here](https://su
...
 | `apiSecret`                            | Your unique api secret.  The `apiSecret` is used as the **password** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                     |  `j54935lddasl837592356aasdf8359hlo3` |
+| `applicationId` | The ID of your messaging application | `532qd-fk5odk5-dlslka40-l5k3lsdmc` |
+
+## Authentication on each API
+
+### Voice
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `userId`                             | Used as a URL parameter for the API endpoint `https://voice.bandwidth.com/v2/account/{userId}`   |
```
Comment (Aubron) [inline accountCredentials.md]: Is this the actual v2 voice URL Structure? I thought userId was scoped to messaging.

Comment (jmulford-bandwidth) [reply accountCredentials.md]: I think it should be `accountId`, I'll update it real quick

Comment (jmulford-bandwidth) [reply accountCredentials.md]: (Based on https://s3.amazonaws.com/slingshot-docs/methods/restApi.html)

Comment (jmulford-bandwidth) [reply accountCredentials.md]: Fixed

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-499.s3-website-us-east-1.amazonaws.com/

File: accountCredentials.md [hunk 2]
```diff
@@ -17,6 +10,40 @@ Specific information for finding your credentials can be found [here](https://su
...
-| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as your accountId. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
+| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as `accountId`. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
```
Comment (dtolb) [inline accountCredentials.md]: No need to use `userId` anymore; in a v2 world it's only account id

File: accountCredentials.md [hunk 3]
```diff
@@ -17,6 +10,40 @@ Specific information for finding your credentials can be found [here](https://su
...
+|:---------------------------------------|:-------------------|
+| `accountId`                             | Used as a URL parameter for the API endpoint `https://voice.bandwidth.com/v2/account/{accountId}`   |
+| `apiToken`                             | Used as the username for HTTP basic auth  | 
+| `apiSecret`                             | Used as the password for HTTP basic auth  | 
+
+### Messaging
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `userId`                             | Used as a URL parameter for the API endpoint `https://messaging.bandwidth.com/api/v2/{userId}/messages`   |
```
Comment (dtolb) [inline accountCredentials.md]: same as above, v2 world uses account id... also endpoint is `https://messaging.bandwidth.com/api/v2/users/{accountId}/messages` ya missed the `users`...

we know that `/users/accountId` is awkward. 🐢

File: accountCredentials.md [hunk 4]
```diff
@@ -17,6 +10,40 @@ Specific information for finding your credentials can be found [here](https://su
...
+|:---------------------------------------|:-------------------|
+| `accountId`  | Used as a URL parameter for the API endpoint `https://dashboard.bandwidth.com/api/accounts/{accountId}`   |
+| `username` | Used as the username for HTTP basic auth  | 
+| `password`  | Used as the password for HTTP basic auth  | 
+
+## Guides for finding and creating credentials
+* [Token and Secret](https://support.bandwidth.com/hc/en-us/articles/360014110974)
+* [Application ID](https://support.bandwidth.com/hc/en-us/articles/360013007654)
+
+<br><br>
```
Comment (dtolb) [inline accountCredentials.md]: needed?

**Review by dtolb — CHANGES_REQUESTED** [review]

File: accountCredentials.md (hunk 2 above)
Comment (jmulford-bandwidth) [reply accountCredentials.md]: Done 👍

File: accountCredentials.md (hunk 3 above)
Comment (jmulford-bandwidth) [reply accountCredentials.md]: Bad copy paste from me 👎 

Fixed 👍

File: accountCredentials.md (hunk 4 above)
Comment (jmulford-bandwidth) [reply accountCredentials.md]: Without these breaks the bottom of the page looks like this. People have commented in the past that it looks like information is cut off

Screen Shot 2019-06-19 at 9 25 20 AM

Comment (jmulford-bandwidth) [reply accountCredentials.md]: Maybe 2 `<br>` tags was too much. I changed it to 1. This is how it looks
Screen Shot 2019-06-19 at 9 27 23 AM

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-499.s3-website-us-east-1.amazonaws.com/

File: accountCredentials.md [hunk 5]
```diff
@@ -17,6 +10,39 @@ Specific information for finding your credentials can be found [here](https://su
...
+
+### Numbers
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `accountId`  | Used as a URL parameter for the API endpoint `https://dashboard.bandwidth.com/api/accounts/{accountId}`   |
+| `username` | Used as the username for HTTP basic auth  | 
+| `password`  | Used as the password for HTTP basic auth  | 
+
+## Guides for finding and creating credentials
```
Comment (dtolb) [inline accountCredentials.md]: Whats you're thought to moving this up top, I think immediately directing people to these links above the fold would be nice.

Comment (jmulford-bandwidth) [reply accountCredentials.md]: I think that's better. Done 👍

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-499.s3-website-us-east-1.amazonaws.com/

**Pull request merged**